src/m365/base/ContextCommand.ts

```diff
@@ -0,0 +1,37 @@
+import * as fs from 'fs';
+import AnonymousCommand from './AnonymousCommand';
+import { Logger } from '../../cli/Logger';
+import { M365RcJson } from './M365RcJson';
+
+export default abstract class ContextCommand extends AnonymousCommand {
+  public saveContextInfo(contextInfo: any, logger: Logger): Promise<any> {
```

Comment:
Since we're passing contextInfo as an argument and not modifying it, is there a reason we're returning it?

Reply:
Not really, this approach was based on the `saveAppInfo` in the app get command. You want met to make saveContextInfo a void function?

Reply:
Let's make it void indeed. If we ever need to get the value back, we can always extend it.

Reply:
Since it is a void now, i didn't really know how to test this in de the context-init.spec.ts file. I wrote a deepStrictEqual test with undefined since it doesn't return anything. But I don't know if this is the right way?

Reply:
The goal of this method is to persist the context in a file, so in the test we should check if that's being done